src/neural/backends/network_onnx.cc

```diff
@@ -331,8 +332,10 @@ Ort::SessionOptions OnnxNetwork::GetOptions(int gpu, int threads,
       trt_options["trt_max_partition_iterations"] = "1000";
       trt_options["trt_min_subgraph_size"] = "1";
       trt_options["trt_engine_cache_enable"] = "1";
+      // We need the batch size as well as the hash, as it is set after loading.
       trt_options["trt_engine_cache_prefix"] =
-          "Lc0_ONNX_TRT_batch_" + std::to_string(batch_size) + "_";
+          "Lc0_ONNX_TRT_ORT_" + Ort::GetVersionString() + "_batch_" +
```

Comment:
There is now `absl::StrCat`, which is usually nicer. It does std::to_string() for non-string parameters automatically, and also faster (not that we care about it here).

There's also `absl::StrFormat`,probably less convenient in this case.